### ISSUE TITLE
Bad java-buildpack v4.20 shipped with cf-deployment 11.0.0

### ISSUE THREAD

**Issue by stephanme** [cloudfoundry/cf-deployment#814]
### What is this issue about?

The java-buildpack v4.20 version shipped as part of cf-deployment v11.0.0 differs from the real v4.20: https://github.com/cloudfoundry/java-buildpack/releases/tag/v4.20

### What version of [cf-deployment](https://github.com/cloudfoundry/cf-deployment/releases) are you using?

cf-deployment v11.0.0 which ships the java-buildpack v4.20 via bosh release https://bosh.io/releases/github.com/cloudfoundry/java-buildpack-release?version=4.20

### Please include the `bosh deploy...` command, including all the operations files (plus any experimental operation files you're using):

n/a

### Please provide output that helps describe the issue:

When staging a java app using the java system buildpack (installed via cf-deployment) I see the following logs:

```
 Downloaded app package (86.7M)
   -----> Java Buildpack v4.20 | https://github.com/cloudfoundry/java-buildpack.git#2cd7e3e
```
Note that the commit sha differs from the one tagged as v4.20 on github (456ce5ea1e90db18b19fd2a811c598a103ba9c65).

### What IaaS is this issue occurring on?

all

### Is there anything else unique or special about your setup?

I also downloaded  https://bosh.io/d/github.com/cloudfoundry/java-buildpack-release?v=4.20 and unpacked it.

spring_auto_reconfiguration.yml contained in the bosh release contains an unspecified version:
```
version: 2.+
repository_root: "{default.repository.root}/auto-reconfiguration"
enabled: true
```

The real v4.20 contains a fixed version 2.7.0_RELEASE: https://github.com/cloudfoundry/java-buildpack/blob/v4.20/config/spring_auto_reconfiguration.yml

**Comment by stephanme** [comment]
https://github.com/cloudfoundry/java-buildpack/releases/tag/v4.19.1 = 0abc75702bddea14c3fa8e01b73b197215c85670 also differs from the version contained in https://bosh.io/releases/github.com/cloudfoundry/java-buildpack-release?version=4.19.1 

version.xml contained in package java-buildpack-cflinuxfs3.tgz
```
---
hash: 3f4eee2
remote: https://github.com/cloudfoundry/java-buildpack.git
version: v4.19.1
```

But the spring_auto_reconfiguration version is fixed on 2.7.0_RELEASE.

**Comment by davewalter** [comment]
As discussed on [Slack](https://cloudfoundry.slack.com/archives/C2U7KA7M4/p1566369745038400), this is an issue with the Java buildpack itself, not with cf-deployment. If a new version of the buildpack is required to fix any issues found, we will automatically include it in the next release.
